docs/administration/upgrade/v6.0_changes.adoc

```diff
@@ -662,12 +687,12 @@ spec:
     outputRefs:
     - es-output-by-label
 ----
-. `es-output-by-label` is the output used in both pipelines
-. `index` is set to read the value from `.openshift.label.my-log-index` and prepend to the string "-write" or fallback to "app-write"
-. `my-app-label` filter is used to set the label "my-log-index:app" in the pipeline
-. `my-infra-label` filter is used to set the label "my-log-index:infra" in the pipeline
-. pipeline `my-app` includes application logs and labels them `app`
-. pipeline `my-infra` includes infrastructure logs and labels them `infra`
+<1> `es-output-by-label` is the output used in both pipelines
+<2> `index` is set to read the value from `.openshift.label.my-log-index` and prepend to the string "-write" or fallback to "app-write"
+<3> `my-app-label` filter is used to set the label "my-log-index:app" in the pipeline
```

Comment:
I would recommend altering the representation of the label to not use the colon; that is not a pattern or syntax I have seen else where. Maybe call it out like key to value or use an equal sign